### DISCUSSION TITLE
Backup and recovery test on GitHub

### DISCUSSION THREAD

**Discussion by KeisukeSuginoshita** [community/community#81807] [New to GitHub]
Hello,

I'm new to GitHub and using it for the first time at my company. There are some things I'd like to check to comply with the company's policies. Specifically, I want to confirm if the GitHub provider conducts backups for disaster recovery and other remote measures, including an annual recovery test. How can I obtain information on this?

I've checked GitHub documentation, but couldn't find specific details. Could you please inform me if there is any information on GitHub regarding the backup and recovery test of the cloud service?

Thank you for your assistance.

**Comment by harrylal** [comment]
Hi @KeisukeSuginoshita 
1.  [Automated backups can be configured to protect production data on your GitHub Enterprise Server instance ](https://docs.github.com/enterprise/admin/guides/installation/backups-and-disaster-recovery)
 2. [In addition, there are third-party backup vendors like GitProtect.io that offer disaster recovery software for GitHub, GitLab, Bitbucket, and Jira ](https://blog.gitguardian.com/github-restore-and-disaster-recovery-better-get-ready/)[2](https://blog.gitguardian.com/github-restore-and-disaster-recovery-better-get-ready/)[3](https://gitprotect.io/blog/github-restore-and-github-disaster-recovery-scenarios-and-use-cases/).
